packages/jellyfish-api-core/__tests__/category/wallet/getBalances.test.ts

```diff
@@ -0,0 +1,87 @@
+import { MasterNodeRegTestContainer, RegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BigNumber } from '../../../src'
```

Comment:
Kindly change the code like what I wrote in your other PR

Reply:
```suggestion
import BigNumber from 'bignumber.js'
```

> https://github.com/DeFiCh/jellyfish/pull/395/files